apps/sms/js/utils.js

```diff
@@ -299,17 +299,17 @@
       if (size < 1000) {
         return {
           l10nId: 'attachmentSizeB',
-          l10nArgs: { n: size }
+          l10nArgs: { n: size, unit: 'byteUnit-B' }
         };
       } else if (sizeKB < 1000) {
         return {
           l10nId: 'attachmentSizeKB',
-          l10nArgs: { n: sizeKB.toFixed(1) }
+          l10nArgs: { n: sizeKB.toFixed(1), unit: 'byteUnit-KB' }
         };
       }
       return {
         l10nId: 'attachmentSizeMB',
-        l10nArgs: { n: sizeMB.toFixed(1) }
+        l10nArgs: { n: sizeMB.toFixed(1), unit: 'byteUnit-MB' }
       };
     },
 
```

Comment:
make sure to update the method comment as well. Currently it says:
```javascript
    /**
    * Gets localization details for attachment size label.
    * @param size Attachment blob size in bytes.
    * @returns {{l10nId: string, l10nArgs: {n: string}}}
    */
```
It should be (notice the alignment, while you’re at it):
```javascript
    /**
     * Gets localization details for attachment size label.
     * @param size Attachment blob size in bytes.
     * @returns { { l10nId: string, l10nArgs: { n: string, unit: string } } }
     */
```